steps/test_30_user_registration.py

```diff
@@ -77,7 +92,7 @@ def user_gets_id(full_name: str, fixtures: JointFixture, response: Response):
     user_id = user["id"]
     assert user_id and "-" in user_id and len(user_id) > 6 and "@" not in user_id
     assert user["name"] == name
-    assert user["title"] == title
+    assert user["title"] == title  # FIXME: title is None, but should be "Dr."
```

Comment:
```suggestion
    assert user["title"] == title  # FIXME: title is None, but should be "Dr."
```

Is this comment still relevant?